tests/unit/router/routes/modelizerRoute.spec.js

```diff
@@ -0,0 +1,49 @@
+import { modelizerRoute } from 'src/router/routes/modelizerRoutes';
+
+describe('Test routes: modelizerRoutes', () => {
+  describe('Test route: modelizerRoute', () => {
+    it('should succeed and call next without paramaters', () => {
+      const next = jest.fn();
+      const to = {
+        params: {
+          projectName: 'projectNameTest',
+          viewType: 'model',
+        },
+      };
+      const projects = { projectNameTest: { id: 1 } };
+      window.localStorage.setItem('projects', JSON.stringify(projects));
+
+      modelizerRoute.beforeEnter(to, null, next);
+      expect(next).toHaveBeenCalledTimes(1);
+      expect(next).toHaveBeenLastCalledWith();
+    });
+
+    it('should error and call next with error page', () => {
```

Comment:
can you separate this test in two:
- Should fail when projet name not exists
- Should fail when viewType is not valid